lib/generators/setup_generator.rb

```diff
@@ -56,9 +55,14 @@ def facets
     return "Customize your facets in app/models/facets.rb".green
   end
 
-  def gems
+  def footer_logo
```

Comment:
oh, lol, now I get why it was above :)

Which solution do you prefer?  Adding the favicon and footer logo to the new app like this?  Or adding them to the precompile step?